provider/datagen/src/transform/segmenter/lstm.rs

```diff
@@ -189,12 +189,12 @@ impl DataProvider<LstmForWordLineAutoV1Marker> for DatagenProvider {
     ) -> Result<DataResponse<LstmForWordLineAutoV1Marker>, DataError> {
         self.check_req::<LstmForWordLineAutoV1Marker>(req)?;
 
-        let model = crate::lstm_data_locale_to_model_name(req.locale)
-            .ok_or(DataErrorKind::MissingLocale.with_req(LstmForWordLineAutoV1Marker::KEY, req))?;
```

Comment:
This seems like a good place to perform the mapping. Maybe if a long name isn't found, we fall back to the long name? This way it could be easier for customers to add custom models.